packages/app/src/app/overmind/namespaces/files/actions.ts

```diff
@@ -22,64 +21,6 @@ import * as internalActions from './internalActions';
 
 export const internal = internalActions;
 
-export const applyRecover = (
-  { state, effects, actions }: Context,
-  recoveredList: Array<{
-    module: Module;
-    recoverData: RecoverData;
-  }>
-) => {
-  if (!state.editor.currentSandbox) {
-    return;
-  }
-
-  effects.moduleRecover.clearSandbox(state.editor.currentSandbox.id);
-  recoveredList.forEach(({ recoverData, module }) => {
-    actions.editor.codeChanged({
-      moduleShortid: module.shortid,
-      code: recoverData.code,
-    });
-    effects.vscode.setModuleCode(module);
-  });
-
-  effects.analytics.track('Files Recovered', {
```

Comment:
Side note, I'd be interested in knowing how many times we recorded this happening, because I didn't see this action much in the little bit of investigation I did on the lives errors in Sentry.